dallinger/recruiters.py

```diff
@@ -278,7 +278,8 @@ def open_recruitment(self, n=1):
 
         self.mturkservice.check_credentials()
 
-        self._create_mturk_qualifications()
+        if self.config.get('assign_qualifications'):
```

Comment:
This should probably default to True for backwards compatibility.

Reply:
Also, this just makes sure that the qualifications are defined. We also need to avoid assigning them to workers in `notify_using`, right?

Reply:
@david: I'm never entirely confident about how config defaults should be implemented. I added a default value of True to `local_config_defaults.txt`, which, as I understand it, should always wind up in the loaded config if nothing of higher precedence is set. Adding a default value to the `get()` feels like I don't quite trust or understand the configuration mechanism (which may be true!) and am doubling up to be safe.  Thoughts?

Reply:
Fixed, thanks.